releasenotes/notes/python2-2_7_18-bump-b5aa063c56926564.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Upgrading the python2 to the latest, and final, 2.7.18 release.
```

for consistency with other release notes :) :

```suggestion
    Upgrading the embedded Python 2 to the latest, and final, 2.7.18 release.
```